### PR TITLE
Customize MQTT Client ID Using A Parameter

### PR DESCRIPTION
**Pull request by mgiannoni**
Use the "client_id" parameter to set the MQTT Client ID.

>             try:
>                 self.client_id = f"{options['client_id']}-{random.randint(0, 1000)}"
>             except:
>                 self.client_id = f'pywis-pubsub-{random.randint(0, 1000)}'

### CODE REVIEW

File: pywis_pubsub/mqtt.py [hunk 1]
```diff
@@ -53,6 +53,10 @@ def __init__(self, broker: str, options: dict = {}) -> None:
 
         if options:
             self.userdata = deepcopy(options)
+            try:
```
Comment (tomkralidis) [inline pywis_pubsub/mqtt.py]: Suggest:

```python
        client_id_prefix = options.get('client_id', 'pywis-pubsub')
        self.client_id = f'{client_id_prefix}-{random.randint(0, 1000)}'
```

File: pywis_pubsub/subscribe.py [hunk 2]
```diff
@@ -183,7 +183,7 @@ def subscribe(ctx, config, download, bbox=[], verbosity='NOTSET'):
     config = util.yaml_load(config)
 
     broker = config.get('broker')
-    default_client_id = f'pywis-pubsub-{random.randint(0, 1000)}'
+    default_client_id = f'pywis-pubsub'
```
Comment (tomkralidis) [inline pywis_pubsub/subscribe.py]: Suggest:

```python
        client_id_prefix = config.get('client_id', 'pywis-pubsub')
        client_id = f'{client_id_prefix}-{random.randint(0, 1000)}'
```

**Review by tomkralidis — CHANGES_REQUESTED** [review]

File: pywis_pubsub/mqtt.py (hunk 1 above)
Comment (mgiannoni) [reply pywis_pubsub/mqtt.py]: Better, yes.  Thanks Tom

File: pywis_pubsub/subscribe.py (hunk 2 above)
Comment (mgiannoni) [reply pywis_pubsub/subscribe.py]: Better again, thanks Tom.

**Comment by mgiannoni** [comment]
Tom:
Made the suggested changes discussed June 27th.
Ready for your review.

Thanks
Marc G

**Review by tomkralidis — APPROVED** [review]

**Comment by tomkralidis** [comment]
Thanks @mgiannoni!

**Pull request merged**